package.json

```diff
@@ -11,6 +11,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "config": "^3.3.6",
```

Comment:
Are we using this anywhere? I'm not sure if we need it

Reply:
I was getting an error so I installed it when I was troubleshooting